Update getting-started.rst (#519)
MetOffice/CSET: docs/source/contributing/getting-started.rst

need to rerun the ``conda activate cset-dev`` command.

.. _conda: https://docs.conda.io/en/latest/

Updating tooling
------------------

If you have been running with your conda environment for some time it might be worth

> re-installing conda and recreate it again from newer lock files.

Reviewer: ```suggestion
recreating your it from newer lock files to get the latest tools.
```

.. code-block:: bash

    # List all your conda environments
    conda info --envs

>     # Remove old conda environments.
>     conda remove -n cset-dev --all -y
> 
> Then follow steps under Setting up tooling again.

Reviewer: ```suggestion
    # Rebuild the conda environment, replacing the existing one.
    conda create -n cset-dev --file requirements/locks/py312-lock-linux-64.txt

Then follow steps under Setting up tooling again.
```
